### PR TITLE
UI-Added Login and CORS

### PR DESCRIPTION
**Pull request by RafaelEPimentel**
-Added the login, register and initiated the navigation bar.
-Added CORS configuration to backend to make API calls from the front-end in development

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (alancuriel) [inline .vscode/launch.json]: .vscode files should not be included in repo

File: financial-tracker-front-end/package.json [hunk 2]
```diff
@@ -16,6 +17,13 @@
...
     "react-dom": "18.2.0",
-    "tailwindcss": "3.3.2"
+    "tailwindcss": "3.3.2",
+    "rxjs": "7.8.1",
+    "swr": "2.2.0",
+    "react-hook-form": "7.45.2",
+    "@hookform/resolvers": "3.1.1",
+    "yup": "1.2.0",
+    "@tailwindcss/forms": "0.5.4",
+    "framer-motion": "10.13.1"
```
Comment (alancuriel) [inline financial-tracker-front-end/package.json:27]: Are we using all of these?

**Review by alancuriel — CHANGES_REQUESTED** [review]

**Review by alancuriel — APPROVED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (RafaelEPimentel) [reply .vscode/launch.json]: removed .vscode

File: financial-tracker-front-end/package.json (hunk 2 above)
Comment (RafaelEPimentel) [reply financial-tracker-front-end/package.json:27]: yes

**Pull request merged**